dbms/src/Flash/Mpp/ExchangeReceiver.cpp

```diff
@@ -204,7 +204,10 @@ class AsyncRequestHandler : public UnaryCallback<bool>
         }
         case AsyncRequestStage::WAIT_BATCH_READ:
             if (ok)
+            {
                 ++read_packet_index;
+                packets[read_packet_index - 1]->recomputeTrackedMem();
```

Comment:
`recomputeTrackedMem` may throw error, we need handle the exception properly.

Reply:
Yes, I move the `try/catch` part into `recomputeTrackedMem `